src/c_kzg_4844.c

```diff
@@ -1377,34 +1402,27 @@ static C_KZG_RET compute_r_powers(
  * Helper function for verify_blob_kzg_proof_batch(): actually perform the
  * verification.
  *
- * @remark This function assumes that `n` is trusted and that all input arrays
- *     contain `n` elements. `n` should be the actual size of the arrays and not
+ * @remark This function assumes that `n` is trusted and that the input array
+ *     contains `n` elements. `n` should be the actual size of the array and not
  *     read off a length field in the protocol.
  *
  * @remark This function only works for `n > 0`.
  *
  * @param[out] ok             True if the proofs are valid, otherwise false
- * @param[in]  commitments_g1 Array of commitments to verify
- * @param[in]  zs_fr          Array of evaluation points for the KZG proofs
- * @param[in]  ys_fr          Array of evaluation results for the KZG proofs
- * @param[in]  proofs_g1      Array of proofs used for verification
+ * @param[in]  claims         Array of claims, each consisting of a commitment,
```

Comment:
You don't need to repeat what a claim consists of here for the same reason I mentioned in the other comment.